include/mbedtls/ssl.h

```diff
@@ -3888,6 +3888,15 @@ size_t mbedtls_ssl_get_bytes_avail( const mbedtls_ssl_context *ssl );
  */
 uint32_t mbedtls_ssl_get_verify_result( const mbedtls_ssl_context *ssl );
 
+/**
+ * \brief          Return the id of the current ciphersuite
+ *
+ * \param ssl      SSL context
+ *
+ * \return         a ciphersuite id
+ */
+int mbedtls_ssl_get_ciphersuite_id_from_ssl( const mbedtls_ssl_context *ssl );
```

Comment:
Following how other accessors are named:
```suggestion
int mbedtls_ssl_get_ciphersuite_id( const mbedtls_ssl_context *ssl );
```

Reply:
@AndrzejKurek `mbedtls_ssl_get_ciphersuite_id` unfortunately already exists and is incompatible with the parameter of the convention you are suggesting.  Some alternatives were discussed in #5417 
https://github.com/ARMmbed/mbedtls/pull/5417#issuecomment-1010627906
https://github.com/ARMmbed/mbedtls/pull/5417#issuecomment-1011580474
https://github.com/ARMmbed/mbedtls/pull/5417#issuecomment-1011612824

Reply:
Please let me know if you would like the accessor name changed.